gen3-client/g3cmd/auth.go

```diff
@@ -0,0 +1,58 @@
+package g3cmd
+
+import (
+	"fmt"
+	"log"
+	"sort"
+
+	"github.com/spf13/cobra"
+	"github.com/uc-cdis/gen3-client/gen3-client/jwt"
+)
+
+func init() {
+
+	var authCmd = &cobra.Command{
+		Use:     "auth",
+		Short:   "Return data access priveleges from profile",
+		Long:    `Gets data access priveleges for specified profile.`,
+		Example: `./gen3-client auth --profile=<profile-name>`,
+		Run: func(cmd *cobra.Command, args []string) {
+
+			request := new(jwt.Request)
+			configure := new(jwt.Configure)
+			function := new(jwt.Functions)
+
+			function.Request = request
+			function.Config = configure
+
+			endPointPostfix := "/user/user" // Information about current user
+
+			host, project_access, err := function.CheckPrivileges(profile, "", endPointPostfix, "application/json", nil)
+
+			if err != nil {
+				log.Fatalf("Fatal authentication error: %s\n", err)
+			} else {
+				if len(project_access) == 0 {
+					fmt.Printf("\nYou don't currently have access to data from any projects at %s\n", host)
+				} else {
+					fmt.Printf("\nYou have access to the following project(s) at %s:\n", host)
+
+					// Sort by project name
+					projects := make([]string, 0, len(project_access))
+					for project := range project_access {
+						projects = append(projects, project)
+					}
+					sort.Strings(projects)
```

Comment:
Very nice here! Can we also sort for each premission in `project_access`?

Reply:
Good idea! Implemented in commit [8cc9275] (https://github.com/uc-cdis/cdis-data-client/pull/49/commits/8cc92755aa94a9dbeb0c12e01c894a06ecf58271).